include/zephyr/drivers/virtualization/ivshmem.h

```diff
@@ -10,6 +10,7 @@
 /**
  * @brief ivshmem reference API
  * @defgroup ivshmem ivshmem reference API
```

Comment:
The lower case "i" is causing this to be put at the end of the alphabetical list of services + it would be good to spell out ivshm too?

```suggestion
 * @brief ivshm reference API.
 * @defgroup ivshmem Inter-VM Shared Memory (ivshmem) reference API
```

Reply:
so should we use `ivshm` or `ivshmem` abbr?

Reply:
sorry, `ivshm` is totally a typo on my side, which I introduced after having to play a couple time with brief and defgroup to understand which was going where in the generated doc :) Sorry!